src/game/tests/test_game.py

```diff
@@ -9,13 +9,13 @@ def setUp(self) -> None:
         self.game = Game()
 
     def test_no_players(self):
-        self.assertRaises(EmptyGameException, self.game.pregame_setup)
+        self.assertRaises(EmptyGameException, self.game._pregame_setup)
```

Comment:
You probably want to add a test with only one player since the boundary value changed